src/App.tsx

```diff
@@ -1,222 +1,234 @@
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React, { useContext, useEffect, useRef } from 'react';
+import React, {
+  useContext,
+  useEffect,
+  useRef,
+  useState,
+} from 'react';
+import {
+  changeTodoStatus,
+  createTodo,
+  deleteTodo,
+  getTodos,
+} from './api/todos';
 import { AuthContext } from './components/Auth/AuthContext';
+import { ErrorNotification } from './components/ErrorNotification';
+import { Footer } from './components/Footer/Footer';
+import { NewTodo } from './components/NewTodo/NewTodo';
+import { TodoList } from './components/Todo';
+import { Todo } from './types/Todo';
+import { TodosFilter } from './types/TodosFilter_Enum';
 
 export const App: React.FC = () => {
   // eslint-disable-next-line @typescript-eslint/no-unused-vars
   const user = useContext(AuthContext);
   const newTodoField = useRef<HTMLInputElement>(null);
 
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [filterTodos, setFilterTodos] = useState<TodosFilter>(TodosFilter.All);
+
+  const [alertText, setAlertText] = useState('');
+  const [isAlertVisible, setAlertVisible] = useState(false);
```

Comment:
You could leave in App only this state about `error` and use it to indicate is you should show error. Other logic move to `ErrorNotification`.